docs/api/browser-window.md

```diff
@@ -1154,10 +1154,18 @@ Returns `boolean` - Whether the window's document has been edited.
 * `opts` Object (optional)
   * `stayHidden` boolean (optional) -  Keep the page hidden instead of visible. Default is `false`.
   * `stayAwake` boolean (optional) -  Keep the system awake instead of allowing it to sleep. Default is `false`.
+  * `outputSize` [Size](structures/size.md) (optional) - The desired width and height in pixels (positive numbers) of the returned image.
 
 Returns `Promise<NativeImage>` - Resolves with a [NativeImage](native-image.md)
 
-Captures a snapshot of the page within `rect`. Omitting `rect` will capture the whole visible page. If the page is not visible, `rect` may be empty. The page is considered visible when its browser window is hidden and the capturer count is non-zero. If you would like the page to stay hidden, you should ensure that `stayHidden` is set to true.
+Captures a snapshot of the page within `rect`. Omitting `rect` will capture the whole visible page.
+If the page is not visible, `rect` may be empty. The page is considered visible when its browser window is hidden and the capturer count is non-zero.
+If you would like the page to stay hidden, you should ensure that `stayHidden` is set to true.
+
+The size of returned [NativeImage](native-image.md) will be `outputSize` if it was provided.
+Otherwise, the size would be large enought to fit all of the available pixels in the current system.
```

Comment:
This is confusing, what does "all of the available pixels in the current system" mean?

How does this behave if the aspect ratio of `outputSize` differs from the aspect ratio of `rect`?

Reply:
For "all of the available pixels":
Initially, I took this phrase somewhere from Chromium sources, but I agree it sounds confusing.

I rephrased it to state the default value instead:

> The desired width and height in pixels (positive numbers) of the returned image. Default is the size of WebContents in physical pixels, that is WebContents height and WebContents width multiplied by device pixel ratio.

For aspect ratio:

I've added a note:
> **Note:** When using `outputSize` it is responsibility of caller to preserve aspect ratio. If aspect ratio of `outputSize` is not the same as page aspect ratio the output image will be scaled.